integration-cli/docker_api_stats_test.go

```diff
@@ -231,6 +231,9 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
 	testRequires(c, DaemonIsLinux)
```

Comment:
> Check out https://github.com/docker/docker/blob/master/integration-cli/requirements_unix.go , it looks like you can just add one of the memory ones to the testRequires here.

Thanks Chris! "memoryLimitSupport" is already defined in requirements_unix.go. So the patch would simply look like the following (seems to work):

```
diff --git a/integration-cli/docker_api_stats_test.go b/integration-cli/docker_api_stats_test.go
index 0e2aa00..a1e4c8b 100644
--- a/integration-cli/docker_api_stats_test.go
+++ b/integration-cli/docker_api_stats_test.go
@@ -229,7 +229,7 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 }
 
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
-       testRequires(c, DaemonIsLinux)
+       testRequires(c, DaemonIsLinux, memoryLimitSupport)
 
        resp, body, err := sockRequestRaw("GET", "/info", nil, "application/json")
        c.Assert(err, checker.IsNil)
```